2_process/src/reclassify_land_cover.R

```diff
@@ -38,3 +39,40 @@ reclassify_land_cover <- function(land_cover_df,
   return(final_df)
   
 }
+
+## -- Specific function for the NLCD reclassifying NLCD land cover classes
+
+reclassify_LC_for_NLCD <- function(NLCD_lc_proportions_df,
+                                   years_suffix,
+                                   reclassify_table_csv_path = '1_fetch/in/Legend_NLCD_Land_Cover.csv'){
+
+  #' @description placing the process of reclassifying the NLCD (2000 +) land cover df into this tailored function. Works with NLCD catchment att Cat and Tot)
```

Comment:
Does it also work for ACC?

```suggestion
  #' @description placing the process of reclassifying the NLCD (2000 +) land cover df into this tailored function. Works with NLCD catchment att CAT and TOT)
```

Reply:
yes